modules/web/js/ballerina/ast/node.js

```diff
@@ -474,35 +474,64 @@ class ASTNode extends EventChannel {
             this.trigger('after-remove');
         }
     }
-
+    /**
+     * Sets isBreakpoint atribute to node
+     *
+     */
     addBreakpoint() {
         this.isBreakpoint = true;
     }
-
+    /**
+     * Removes isBreakpoint atribute to node
+     *
+     */
     removeBreakpoint() {
         this.isBreakpoint = false;
     }
-
+    /**
+     * Sets isDebugHit atribute to node
+     *
+     */
     addDebugHit() {
         this.setAttribute('isDebugHit', true);
     }
-
+    /**
+     * Removes isDebugHit atribute from node
+     *
+     */
     removeDebugHit() {
         this.setAttribute('isDebugHit', false);
     }
-
+    /**
+     * Set lineNumber atribute to node
+     * @param {Number} lineNumber
```

Comment:
If the param is a primitive(number) type, we have to use 'number' instead of 'Number'